tests/generate_packageTest.py

```diff
@@ -90,25 +91,41 @@ class mainTest(unittest.TestCase):
 
     def setUp(self):
         self.old_argv = sys.argv
+        sys.argv = ["generate_package.py"]
 
     def test_main(self):
-        sys.argv = ["generate_package.py"]
         main()
         self.assertTrue(os.path.exists(os.path.join('bears', 'upload')))
         with open(self.CSS_BEAR_SETUP_PATH) as fl:
             setup_py = fl.read()
         self.assertIn("Check code for syntactical or semantical", setup_py)
 
+    def test_main_bear_version(self):
+        fake_prod_version = '99.99.99'
+        with patch('bears.generate_package.VERSION', fake_prod_version):
+            main()
+        with open(self.CSS_BEAR_SETUP_PATH) as fl:
+            setup_py = fl.read()
+        self.assertIn(fake_prod_version, setup_py)
+        self.assertNotIn(VERSION, setup_py)
+
+    def test_main_bear_version_dev(self):
+        main()
+        with open(self.CSS_BEAR_SETUP_PATH) as fl:
+            setup_py = fl.read()
+        self.assertNotIn(VERSION, setup_py)
+        self.assertIn(VERSION[:VERSION.find("dev")], setup_py)
+
     @patch('bears.generate_package.perform_upload')
     def test_upload(self, call_mock):
-        sys.argv = ["generate_package.py", "--upload"]
+        sys.argv.append("--upload")
```

Comment:
I'm happy to omit moving the argv to setUp, but my new tests also need that set, so it seemed cleaner to move it to setUp and save some LOCs. Please let me know if this seems reasonable?